e2e_tests/tests/nightly/test_distributed.py

```diff
@@ -205,8 +205,8 @@ def test_deepspeed_pipeline_parallel() -> None:
 
 @pytest.mark.deepspeed
 @pytest.mark.gpu_required
-def test_gpt_neox_zero_medium() -> None:
-    config = conf.load_config(conf.deepspeed_examples_path("gpt_neox/zero3_medium.yaml"))
+def test_gpt_neox_zero1() -> None:
+    config = conf.load_config(conf.deepspeed_examples_path("gpt_neox/zero3.yaml"))
```

Comment:
wait, am i missing something? what is the difference now between test_gpt_neox_zero3 and test_gpt_neox_zero1? should we just remove one of the tests?

Reply:
nvm, i think this is a typo. should be zero1.yaml.